app/api/views/briefs.py

```diff
@@ -1195,20 +1107,21 @@ def download_brief_attachment(brief_id, slug):
     if not brief:
         return not_found('File not found')
 
+    user_role = current_user.role if hasattr(current_user, 'role') else None
+    user_status = BriefUserStatus(brief, current_user)
+
     all_documents = (
         brief.data.get('attachments', []) +
         brief.data.get('requirementsDocument', []) +
         brief.data.get('responseTemplate', [])
     )
 
     if (
-        hasattr(current_user, 'role') and
-        (
-            current_user.role == 'buyer' or (
-                current_user.role == 'supplier' and
-                _can_do_brief_response(brief_id, update_only=True) and
-                slug in all_documents
-            )
+        user_role == 'admin' or
```

Comment:
I'm adding this one because previously an admin hasn't been able to download brief attachments.